packages/ui/src/views/repo/pull-request/components/pull-request-diff-viewer.tsx

```diff
@@ -399,10 +399,10 @@ const PullRequestDiffViewer = ({
                 quoteReplyText={quoteReplies[parent.id]?.text || ''}
                 contentHeader={
                   !!parent.payload?.resolved && (
-                    <div className="flex items-center gap-x-1">
+                    <div className="text-14 text-foreground-4">
                       {/* TODO: need to identify the author who resolved the conversation */}
-                      <span className="font-medium text-foreground-8">{parent.author}</span>
-                      <span className="text-foreground-4">marked this conversation as resolved</span>
+                      <span className="font-medium text-foreground-1">{currentUser}</span> marked this conversation as
```

Comment:
shouldn't be currentUser, as if i resolve issue on different account it would use the account that i'd currently be on @andrewgolovanov

Reply:
nvm saw issue on this